src/libsystemd/sd-netlink/netlink-types.c

```diff
@@ -9,6 +9,7 @@
 #include <linux/genetlink.h>
 #include <linux/ip.h>
 #include <linux/if.h>
+#include <linux/batman_adv.h>
```

Comment:
Please sort all headers alphabetically if you're touching this. Nowadays we try to keep them alphabetiacl.

Reply:
It seems like this sorting wasn't addressed yet?